development/openstack-base-focal-yoga/bundle.yaml

```diff
@@ -321,87 +313,80 @@ applications:
     - '0'
     - '1'
     - '2'
-    channel: yoga/edge
-  ntp:
-    annotations:
-      gui-x: '315'
-      gui-y: '1030'
-    charm: ch:ntp
-    num_units: 0
   dashboard-mysql-router:
     annotations:
       gui-x: '510'
       gui-y: '1030'
     charm: ch:mysql-router
-    channel: 8.0.19/edge
+    channel: 8.0.19/stable
   openstack-dashboard:
     annotations:
       gui-x: '585'
       gui-y: '900'
     charm: ch:openstack-dashboard
+    channel: yoga/stable
     num_units: 1
     options:
       openstack-origin: *openstack-origin
     to:
     - lxd:1
-    channel: yoga/edge
   rabbitmq-server:
     annotations:
       gui-x: '300'
       gui-y: '1550'
     charm: ch:rabbitmq-server
+    channel: 3.9/stable
```

Comment:
focal should 3.8/stable
while jammy would 3.9/stable

Reply:
Ack